test/AccrueInterestTest.sol

```diff
@@ -153,14 +190,13 @@ contract AccrueInterestTest is BaseTest {
         vm.stopPrank();
         vault.setPerformanceFee(performanceFee);
         vault.setManagementFee(managementFee);
-        vm.warp(vm.getBlockTimestamp() + elapsed);
-
-        // Rate too high.
         vm.prank(allocator);
         vic.setInterestPerSecondAndDeadline(interestPerSecond, type(uint64).max);
-        uint256 totalAssetsBefore = vault.totalAssets();
```

Comment:
nothing was tested here because of this. I removed the use of totalAssets everywhere in the tests, and wrote a test for it specifically

Reply:
I was looking at this right now, I think that nothing was tested also because in `vic.setInterestPerSecondAndDeadline`, `accrueInterest` is called (with the elapsed skip it should be good now).